mono/metadata/class-accessors.c

```diff
@@ -389,3 +389,11 @@ mono_class_gtd_get_canonical_inst (MonoClass *klass)
 	g_assert (mono_class_is_gtd (klass));
 	return &((MonoClassGtd*)klass)->canonical_inst;
 }
+
+void
+mono_class_set_blittable (MonoClass *klass, gboolean blittable)
```

Comment:
I think this is the wrong level to add locking, it's too fine grained.
There's the expectation that setters don't take lock. The COM one is doing it, but shouldn't.

Besides that, it's missing locking comments.